src/www/views/servers_view/server_list_item/server_card/index.ts

```diff
@@ -188,7 +189,7 @@ const getSharedComponents = (element: ServerListItemElement & LitElement) => {
       metadataText: html`
         <div class="card-metadata-text">
           <h2 class="card-metadata-server-name" id="server-name">
-            ${messages.serverName}
+            ${messages.serverName} Loremipsumdolorsitamet,consecteturadipisicingelit.Ea,suscipit.
```

Comment:
Do you mind removing this placeholder text?

Reply:
sorry, i fixed it